<!doctype html><html lang=en-us><head><link rel=preconnect href=https://fonts.googleapis.com><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@700&display=swap" rel=stylesheet><meta charset=utf-8><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><script src=https://ajax.googleapis.com/ajax/libs/jquery/3.6.1/jquery.min.js></script><script>window.onload=function(){document.getElementById("expand-proj").addEventListener("click",function(){this.textContent=="show less..."?this.textContent="show all...":this.textContent="show less..."}),document.getElementById("expand-uni").addEventListener("click",function(){this.textContent=="show less..."?this.textContent="show all...":this.textContent="show less..."})}</script><script>var unis={};$(document).on("change","input[type='checkbox']",function(){if(this.id.startsWith("check_year")){var e,n,t=$(':input[id^="check_year"]').map(function(){return $(this).prop("checked")}).get().includes(!0);const s=document.getElementById("main").children;if($("#year_list li:first-child").css("display")=="none"&&($("#year_list li:first-child").prop("style","display:initial !important"),$('div[id^="year_"]').prop("style","display:none !important")),t)for(child of s)child.id.startsWith(this.id.slice(-9))&&(this.checked?child.style.display="initial":child.style.display="none");else{$("#year_list li:first-child").prop("style","display:none !important");for(child of s)child.style.display="initial"}}else if(this.id.startsWith("uni"))if(console.log("Hi2"),t=$(':input[id^="uni"]').map(function(){return $(this).prop("checked")}).get().includes(!0),e=this.id.slice(-7),unis[e]==void 0&&(unis[e]=!0),this.checked?unis[e]=!0:unis[e]=!1,$("#university_list li:first-child").css("display")=="none"&&($("#university_list li:first-child").prop("style","display:initial !important"),$('div[data-prj^="prj"]').each(function(){$(this).children().first().prop("style","display:none !important")})),t){const t=document.querySelectorAll("[data-uni="+e+"]");for(child of t)if(this.checked)child.parentNode.parentNode.setAttribute("style","display:flex !important");else{disp_off_flag=!0;for(child of child.parentNode.querySelectorAll("[data-uni]"))if(unis[child.getAttribute("data-uni")]==void 0&&(unis[child.getAttribute("data-uni")]=!0),unis[child.getAttribute("data-uni")]==!0){disp_off_flag=!1;break}disp_off_flag&&child.parentNode.parentNode.setAttribute("style","display:none !important")}}else $("#university_list li:first-child").prop("style","display:none !important"),$('div[data-prj^="prj"]').each(function(){$(this).children().first().prop("style","display:flex !important")});else if(this.id.startsWith("prj"))if(t=$(':input[id^="prj"]').map(function(){return $(this).prop("checked")}).get().includes(!0),n=this.id.slice(4),$("#project_list li:first-child").css("display")=="none"&&($("#project_list li:first-child").prop("style","display:initial !important"),$('div[data-prj^="prj"]').each(function(){$(this).prop("style","display:none !important")})),t){const e=document.querySelectorAll("[data-prj="+n+"]");for(child of e)this.checked?child.setAttribute("style","display:initial !important"):child.setAttribute("style","display:none !important")}else console.log("all unckecked"),$("#project_list li:first-child").prop("style","display:none !important"),$('div[data-prj^="prj"]').each(function(){$(this).prop("style","display:initial !important")})});function clearAll(e){e=="year"?($(':input[id^="check_year"]').prop("checked",!1),$('div[id^="year_"]').prop("style","display:initial;"),$("#year_list li:first-child").prop("style","display:none !important")):e=="uni"?($(':input[id^="uni"]').prop("checked",!1),$('div[data-prj^="prj"]').each(function(){$(this).children().first().prop("style","display:flex !important")}),$("#university_list li:first-child").prop("style","display:none !important")):e=="prj"&&($(':input[id^="prj"]').prop("checked",!1),$('div[data-prj^="prj"]').each(function(){$(this).prop("style","display:initial !important")}),$("#project_list li:first-child").prop("style","display:none !important"))}</script><style>.arrowup{border:solid #fff;border-width:0 3px 3px 0;display:inline-block;padding:5px;transform:rotate(-135deg);-webkit-transform:rotate(-135deg)}#topBtn{display:none;position:fixed;bottom:20px;right:30px;z-index:99;font-size:30px;border:none;outline:none;background-color:#29abe0;color:#fff;cursor:pointer;padding:15px;border-radius:8px;height:10dp}#topBtn:hover{background-color:#1b8dbb}.card-header-custom{cursor:pointer;-webkit-user-select:none;-ms-user-select:none;user-select:none}.clear-widget{color:#747474!important;background-color:#dee2e6;font-size:80%!important}.clear-widget:hover{color:#212529!important;background-color:#c1c9d0;cursor:pointer}</style><!--[if IEMobile]><meta http-equiv=cleartype content="on"><![endif]--><title>Patrick M. D’Aoust - GWF Publications</title>
<meta name=generator content="Hugo 0.124.1"><link href=/gwf-publications/gwficon.ico rel="shortcut icon" type=image/x-icon><link rel=stylesheet href=/gwf-publications/css/main.min.b1d14a9a8f6bb9c608ca4de9aad72a6e06945119f97951f2908522dc220e6277.css media=screen><link rel=stylesheet href=https://use.fontawesome.com/releases/v5.7.2/css/all.css integrity=sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr crossorigin=anonymous><link rel=stylesheet href=/css/academicons.min.css></head><body><header style='display:block;box-sizing:border-box;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;line-height:1.5;color:#222;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%'><div id=navbar-container style='height:54px;width:100%;background:rgba(255,255,255,.95);overflow:hidden;-webkit-box-shadow:rgba(0,0,0,9%)0 1px 0;box-shadow:rgba(0,0,0,9%)0 1px;box-sizing:border-box;display:block;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;line-height:1.5;color:#222;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%'><a href=/ style='color:#0072bc;text-decoration:none;display:inline-block;height:100%;line-height:54px;border-right:1px solid #e6e6e6;margin-left:6px;padding-right:6px;outline-offset:-4px!important;font-weight:700;background:0 0;box-sizing:border-box;cursor:pointer;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;-webkit-tap-highlight-color:transparent'><img src=/images/gwf-logo.png alt="ACL Logo" style='margin-right:0;height:54px;padding:6px;width:auto;margin-top:-2px;max-width:100%;vertical-align:middle;border:0;box-sizing:border-box;color:#0072bc;line-height:54px;font-weight:700;cursor:pointer;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%'>
</a><a href=/ style='text-decoration:none;color:#222!important;display:inline-block;font-size:16px;line-height:54px;font-weight:700;outline-offset:-4px!important;padding:0 6px;margin:0 -6px;background:0 0;box-sizing:border-box;cursor:pointer;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%;padding-left:4px'>Global Water Futures</a></div></header><div id=main-container class=container><section id=main><h2 id=title><span class=font-weight-normal>Patrick M.</span> <span class=font-weight-bold>D’Aoust</span></h2><hr><div class=row><div class=col-lg-9><h4>2023</h4><div data-prj=prj23><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.3389/fmicb.2023.1048661 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/G23-68001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G23-68001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G23-68001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G23-68001/>Realizing the value in “non-standard” parts of the qPCR standard curve by integrating fundamentals of quantitative microbiology</a></strong><br><a href=/gwf-publications/people/P/Philip-J-Schmidt/ data-uni=uni0002>Philip J. Schmidt</a>,
<a href=/gwf-publications/people/N/Nicole-Acosta/ data-uni=uni0007>Nicole Acosta</a>,
<a href=/gwf-publications/people/A/Alex-H-S-Chik/ data-uni=uni0451>Alex H. S. Chik</a>,
<a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=uni0027>Patrick M. D’Aoust</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0027>Robert Delatolla</a>,
<a href=/gwf-publications/people/H/Hadi-A-Dhiyebi/ data-uni=uni0002>Hadi A. Dhiyebi</a>,
<a href=/gwf-publications/people/M/Melissa-B-Glier/ data-uni=uni0771>Melissa B. Glier</a>,
<a href=/gwf-publications/people/C/Casey-R-J-Hubert/ data-uni=uni0007>Casey R. J. Hubert</a>,
<a href=/gwf-publications/people/J/Jennifer-Kopetzky/ data-uni=uni0196>Jennifer Kopetzky</a>,
<a href=/gwf-publications/people/C/Chand-Mangat/ data-uni=uni0107>Chand Mangat</a>,
<a href=/gwf-publications/people/X/Xiaoli-Pang/ data-uni=uni0376>Xiaoli Pang</a>,
<a href=/gwf-publications/people/S/Shelley-Peterson/ data-uni=uni0107>Shelley Peterson</a>,
<a href=/gwf-publications/people/N/Natalie-Prystajecky/ data-uni=uni0924>Natalie Prystajecky</a>,
<a href=/gwf-publications/people/Y/Yuanyuan-Qiu/ data-uni=uni0509>Yuanyuan Qiu</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/M/Monica-B-Emelko/ data-uni=uni0002>Monica B. Emelko</a><br><a href=/gwf-publications/volumes/G23-68/ class=text-muted>Frontiers in Microbiology, Volume 14</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G23-68001><div class="card-body p-3 small">The real-time polymerase chain reaction (PCR), commonly known as quantitative PCR (qPCR), is increasingly common in environmental microbiology applications. During the COVID-19 pandemic, qPCR combined with reverse transcription (RT-qPCR) has been used to detect and quantify SARS-CoV-2 in clinical diagnoses and wastewater monitoring of local trends. Estimation of concentrations using qPCR often features a log-linear standard curve model calibrating quantification cycle (Cq) values obtained from underlying fluorescence measurements to standard concentrations. This process works well at high concentrations within a linear dynamic range but has diminishing reliability at low concentrations because it cannot explain "non-standard" data such as Cq values reflecting increasing variability at low concentrations or non-detects that do not yield Cq values at all. Here, fundamental probabilistic modeling concepts from classical quantitative microbiology were integrated into standard curve modeling approaches by reflecting well-understood mechanisms for random error in microbial data. This work showed that data diverging from the log-linear regression model at low concentrations as well as non-detects can be seamlessly integrated into enhanced standard curve analysis. The newly developed model provides improved representation of standard curve data at low concentrations while converging asymptotically upon conventional log-linear regression at high concentrations and adding no fitting parameters. Such modeling facilitates exploration of the effects of various random error mechanisms in experiments generating standard curve data, enables quantification of uncertainty in standard curve parameters, and is an important step toward quantifying uncertainty in qPCR-based concentration estimates. Improving understanding of the random error in qPCR data and standard curve modeling is especially important when low concentrations are of particular interest and inappropriate analysis can unduly affect interpretation, conclusions regarding lab performance, reported concentration estimates, and associated decision-making.</div></div><h4>2022</h4><div data-prj=prj23><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.scitotenv.2022.158547 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/G22-155001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-155001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-155001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-155001/>Wastewater to clinical case (WC) ratio of COVID-19 identifies insufficient clinical testing, onset of new variants of concern and population immunity in urban communities</a></strong><br><a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=uni0027>Patrick M. D’Aoust</a>,
<a href=/gwf-publications/people/X/Xin-Tian/ data-uni=uni0103>Xin Tian</a>,
<a href=/gwf-publications/people/S/Syeda-Tasneem-Towhid/ data-uni=uni0027>Syeda Tasneem Towhid</a>,
<a href=/gwf-publications/people/A/Amy-Xiao/ data-uni=uni0246>Amy Xiao</a>,
<a href=/gwf-publications/people/%C3%89/%C3%89lisabeth-Mercier/ data-uni=uni0027>Élisabeth Mercier</a>,
<a href=/gwf-publications/people/N/Nada-Hegazy/ data-uni=uni0027>Nada Hegazy</a>,
<a href=/gwf-publications/people/J/Jianjun-Jia/ data-uni=uni0212>Jianjun Jia</a>,
<a href=/gwf-publications/people/S/Shungang-Wan/ data-uni=uni0237>Shungang Wan</a>,
<a href=/gwf-publications/people/M/Md-Pervez-Kabir/ data-uni=uni0027>Md Pervez Kabir</a>,
<a href=/gwf-publications/people/W/Wanting-Fang/ data-uni=uni0027>Wanting Fang</a>,
<a href=/gwf-publications/people/M/Meghan-Fuzzen/ data-uni=uni0002>Meghan Fuzzen</a>,
<a href=/gwf-publications/people/M/Maria-E-Hasing/ data-uni=uni0012>Maria E. Hasing</a>,
<a href=/gwf-publications/people/M/Minqing-Ivy-Yang/ data-uni=uni0022>Minqing Ivy Yang</a>,
<a href=/gwf-publications/people/J/Jianxian-Sun/ data-uni=uni0022>Jianxian Sun</a>,
<a href=/gwf-publications/people/J/Julio-PlazaD%C3%ADaz/ data-uni=uni1074>Julio Plaza‐Díaz</a>,
<a href=/gwf-publications/people/Z/Zhihao-Zhang/ data-uni=uni1075>Zhihao Zhang</a>,
<a href=/gwf-publications/people/A/Aaron-Cowan/ data-uni=uni0027>Aaron Cowan</a>,
<a href=/gwf-publications/people/W/Walaa-Eid/ data-uni=uni0117>Walaa Eid</a>,
<a href=/gwf-publications/people/S/Sean-E-Stephenson/ data-uni=uni0117>Sean E. Stephenson</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/M/Matthew-J-Wade/ data-uni=uni0398>Matthew J. Wade</a>,
<a href=/gwf-publications/people/A/Alex-MacKenzie/ data-uni=uni0117>Alex MacKenzie</a>,
<a href=/gwf-publications/people/H/Hui-Peng/ data-uni=uni0022>Hui Peng</a>,
<a href=/gwf-publications/people/E/Elizabeth-A-Edwards/ data-uni=uni0022>Elizabeth A. Edwards</a>,
<a href=/gwf-publications/people/X/Xiaoli-Pang/ data-uni=uni0376>Xiaoli Pang</a>,
<a href=/gwf-publications/people/E/Eric-J-Alm/ data-uni=uni0246>Eric J. Alm</a>,
<a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=uni0117>Tyson E. Graber</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0027>Robert Delatolla</a><br><a href=/gwf-publications/volumes/G22-155/ class=text-muted>Science of The Total Environment, Volume 853</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-155001><div class="card-body p-3 small">Clinical testing has been the cornerstone of public health monitoring and infection control efforts in communities throughout the COVID-19 pandemic. With the anticipated reduction of clinical testing as the disease moves into an endemic state, SARS-CoV-2 wastewater surveillance (WWS) will have greater value as an important diagnostic tool. An in-depth analysis and understanding of the metrics derived from WWS is required to interpret and utilize WWS-acquired data effectively (McClary-Gutierrez et al., 2021; O'Keeffe, 2021). In this study, the SARS-CoV-2 wastewater signal to clinical cases (WC) ratio was investigated across seven cities in Canada over periods ranging from 8 to 21 months. This work demonstrates that significant increases in the WC ratio occurred when clinical testing eligibility was modified to appointment-only testing, identifying a period of insufficient clinical testing (resulting in a reduction to testing access and a reduction in the number of daily tests) in these communities, despite increases in the wastewater signal. Furthermore, the WC ratio decreased significantly in 6 of the 7 studied locations, serving as a potential signal of the emergence of the Alpha variant of concern (VOC) in a relatively non-immunized community (40-60 % allelic proportion), while a more muted decrease in the WC ratio signaled the emergence of the Delta VOC in a relatively well-immunized community (40-60 % allelic proportion). Finally, a significant decrease in the WC ratio signaled the emergence of the Omicron VOC, likely because of the variant's greater effectiveness at evading immunity, leading to a significant number of new reported clinical cases, even when community immunity was high. The WC ratio, used as an additional monitoring metric, could complement clinical case counts and wastewater signals as individual metrics in its potential ability to identify important epidemiological occurrences, adding value to WWS as a diagnostic technology during the COVID-19 pandemic and likely for future pandemics.</div></div><h4>2021</h4><div data-prj=prj1><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.jes.2021.01.029 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/G21-41001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-41001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-41001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-41001/>Comparison of approaches to quantify SARS-CoV-2 in wastewater using RT-qPCR: Results and implications from a collaborative inter-laboratory study in Canada</a></strong><br><a href=/gwf-publications/people/A/Alex-H-S-Chik/ data-uni=uni0451>Alex H. S. Chik</a>,
<a href=/gwf-publications/people/M/Melissa-B-Glier/ data-uni=uni0771>Melissa B. Glier</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/C/Chand-Mangat/ data-uni=uni0107>Chand Mangat</a>,
<a href=/gwf-publications/people/X/Xiaoli-Pang/ data-uni=uni0376>Xiaoli Pang</a>,
<a href=/gwf-publications/people/Y/Yuanyuan-Qiu/ data-uni=uni0509>Yuanyuan Qiu</a>,
<a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=uni0027>Patrick M. D’Aoust</a>,
<a href=/gwf-publications/people/J/Jean-Baptiste-Burnet/ data-uni=uni0239>Jean-Baptiste Burnet</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0027>Robert Delatolla</a>,
<a href=/gwf-publications/people/S/Sarah-Dorner/ data-uni=uni0239>Sarah Dorner</a>,
<a href=/gwf-publications/people/Q/Qiudi-Geng/ data-uni=uni0153>Qiudi Geng</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0016>John P. Giesy</a>,
<a href=/gwf-publications/people/R/R-Michael-L-McKay/ data-uni=uni0153>R. Michael L. McKay</a>,
<a href=/gwf-publications/people/M/Michael-R-Mulvey/ data-uni=uni0107>Michael R. Mulvey</a>,
<a href=/gwf-publications/people/N/Natalie-Prystajecky/ data-uni=uni0924>Natalie Prystajecky</a>,
<a href=/gwf-publications/people/N/Nivetha-Srikanthan/ data-uni=uni0002>Nivetha Srikanthan</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0050>Yuwei Xie</a>,
<a href=/gwf-publications/people/B/Bernadette-Conant/ data-uni=uni1570>Bernadette Conant</a>,
<a href=/gwf-publications/people/S/Steve-E-Hrudey/ data-uni=uni0012>Steve E. Hrudey</a><br><a href=/gwf-publications/volumes/G21-41/ class=text-muted>Journal of Environmental Sciences, Volume 107</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-41001><div class="card-body p-3 small">Detection of SARS-CoV-2 RNA in wastewater is a promising tool for informing public health decisions during the COVID-19 pandemic. However, approaches for its analysis by use of reverse transcription quantitative polymerase chain reaction (RT-qPCR) are still far from standardized globally. To characterize inter- and intra-laboratory variability among results when using various methods deployed across Canada, aliquots from a real wastewater sample were spiked with surrogates of SARS-CoV-2 (gamma-radiation inactivated SARS-CoV-2 and human coronavirus strain 229E [HCoV-229E]) at low and high levels then provided "blind" to eight laboratories. Concentration estimates reported by individual laboratories were consistently within a 1.0-log10 range for aliquots of the same spiked condition. All laboratories distinguished between low- and high-spikes for both surrogates. As expected, greater variability was observed in the results amongst laboratories than within individual laboratories, but SARS-CoV-2 RNA concentration estimates for each spiked condition remained mostly within 1.0-log10 ranges. The no-spike wastewater aliquots provided yielded non-detects or trace levels (&lt;20 gene copies/mL) of SARS-CoV-2 RNA. Detections appear linked to methods that included or focused on the solids fraction of the wastewater matrix and might represent in-situ SARS-CoV-2 to the wastewater sample. HCoV-229E RNA was not detected in the no-spike aliquots. Overall, all methods yielded comparable results at the conditions tested. Partitioning behavior of SARS-CoV-2 and spiked surrogates in wastewater should be considered to evaluate method effectiveness. A consistent method and laboratory to explore wastewater SARS-CoV-2 temporal trends for a given system, with appropriate quality control protocols and documented in adequate detail should succeed.</div></div><div data-prj=prj1><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1101/2021.02.22.21252041 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/G21-62010.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-62010 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-62010 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-62010/>Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy</a></strong><br><a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=uni0117>Tyson E. Graber</a>,
<a href=/gwf-publications/people/K/Kamya-Bhatnagar/ data-uni=uni0027>Kamya Bhatnagar</a>,
<a href=/gwf-publications/people/%C3%89/%C3%89lisabeth-Mercier/ data-uni=uni0027>Élisabeth Mercier</a>,
<a href=/gwf-publications/people/M/Meghan-Fuzzen/ data-uni=uni0002>Meghan Fuzzen</a>,
<a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=uni0027>Patrick M. D’Aoust</a>,
<a href=/gwf-publications/people/H/Huy-Dung-Hoang/ data-uni=uni0117>Huy-Dung Hoang</a>,
<a href=/gwf-publications/people/X/Xin-Tian/ data-uni=uni0103>Xin Tian</a>,
<a href=/gwf-publications/people/S/Syeda-Tasneem-Towhid/ data-uni=uni0027>Syeda Tasneem Towhid</a>,
<a href=/gwf-publications/people/J/Julio-Plaza-Diaz/ data-uni=uni0117>Julio Plaza Diaz</a>,
<a href=/gwf-publications/people/T/Tommy-Alain/ data-uni=uni0027>Tommy Alain</a>,
<a href=/gwf-publications/people/A/Ainslie-Butler/ data-uni=uni1578>Ainslie Butler</a>,
<a href=/gwf-publications/people/L/Lawrence-Goodridge/ data-uni=uni0009>Lawrence Goodridge</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0027>Robert Delatolla</a><br><a href=/gwf-publications/volumes/G21-62/ class=text-muted></a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-62010><div class="card-body p-3 small">Abstract The coronavirus disease 2019 (COVID-19) pandemic caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has claimed millions of lives to date. Antigenic drift has resulted in viral variants with putatively greater transmissibility, virulence, or both. Early and near real-time detection of these variants of concern (VOC) and the ability to accurately follow their incidence and prevalence in communities is wanting. Wastewater-based epidemiology (WBE), which uses nucleic acid amplification tests to detect viral fragments, is a faithful proxy of COVID-19 incidence and prevalence, and thus offers the potential to monitor VOC viral load in a given population. Here, we describe and validate a primer extension PCR strategy targeting a signature mutation in the N gene of SARS-CoV-2. This allows quantification of the proportional expression of B.1.1.7 versus non-B.1.1.7 alleles in wastewater without the need to employ quantitative RT-PCR standard curves. We show that the wastewater B.1.1.7 profile correlates with its clinical counterpart and benefits from a near real-time and facile data collection and reporting pipeline. This assay can be quickly implemented within a current SARS-CoV-2 WBE framework with minimal cost; allowing early and contemporaneous estimates of B.1.1.7 community transmission prior to, or in lieu of, clinical screening and identification. Our study demonstrates that this strategy can provide public health units with an additional and much needed tool to rapidly triangulate VOC incidence/prevalence with high sensitivity and lineage specificity.</div></div><div data-prj=prj1><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.watres.2020.116560 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/G21-222001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-222001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-222001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-222001/>Quantitative analysis of SARS-CoV-2 RNA from wastewater solids in communities with low COVID-19 incidence and prevalence</a></strong><br><a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=uni0027>Patrick M. D’Aoust</a>,
<a href=/gwf-publications/people/%C3%89/%C3%89lisabeth-Mercier/ data-uni=uni0027>Élisabeth Mercier</a>,
<a href=/gwf-publications/people/D/Danika-Montpetit/ data-uni=uni0004>Danika Montpetit</a>,
<a href=/gwf-publications/people/J/Jian-Jun-Jia/ data-uni=uni0027>Jian Jun Jia</a>,
<a href=/gwf-publications/people/I/Ilya-Alexandrov/ data-uni=uni0802>Ilya Alexandrov</a>,
<a href=/gwf-publications/people/N/Nafisa-Neault/ data-uni=uni0027>Nafisa Neault</a>,
<a href=/gwf-publications/people/A/Aiman-Tariq-Baig/ data-uni=uni0117>Aiman Tariq Baig</a>,
<a href=/gwf-publications/people/J/Janice-Mayne/ data-uni=uni0027>Janice Mayne</a>,
<a href=/gwf-publications/people/X/Xu-Zhang/ data-uni=uni0576>Xu Zhang</a>,
<a href=/gwf-publications/people/T/Tommy-Alain/ data-uni=uni0027>Tommy Alain</a>,
<a href=/gwf-publications/people/M/Marc-Andr%C3%A9-Langlois/ data-uni=uni0602>Marc André Langlois</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/M/Malcolm-R-MacKenzie/ data-uni=uni0022>Malcolm R. MacKenzie</a>,
<a href=/gwf-publications/people/D/Daniel-Figeys/ data-uni=uni0027>Daniel Figeys</a>,
<a href=/gwf-publications/people/A/Alex-MacKenzie/ data-uni=uni0117>Alex MacKenzie</a>,
<a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=uni0117>Tyson E. Graber</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0027>Robert Delatolla</a><br><a href=/gwf-publications/volumes/G21-222/ class=text-muted>Water Research, Volume 188</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-222001><div class="card-body p-3 small">• RT-ddPCR is more sensitive to inhibitors than RT-qPCR for primary clarified sludge. • Primary clarified sludge has elevated frequency of SARS-CoV-2 RNA detection. • Primary clarified sludge allows detection of RNA during low COVID-19 incidence. • PMMoV normalization of RNA data reduces noise and increases precision. • PMMoV normalization of RNA shows strongest correlation to epidemiological metrics. In the absence of an effective vaccine to prevent COVID-19 it is important to be able to track community infections to inform public health interventions aimed at reducing the spread and therefore reduce pressures on health-care, improve health outcomes and reduce economic uncertainty. Wastewater surveillance has rapidly emerged as a potential tool to effectively monitor community infections through measuring trends of RNA signal in wastewater systems. In this study SARS-CoV-2 viral RNA N1 and N2 gene regions are quantified in solids collected from influent post grit solids (PGS) and primary clarified sludge (PCS) in two water resource recovery facilities (WRRF) serving Canada's national capital region, i.e., the City of Ottawa, ON (pop. ≈ 1.1M) and the City of Gatineau, QC (pop. ≈ 280K). PCS samples show signal inhibition using RT-ddPCR compared to RT-qPCR, with PGS samples showing similar quantifiable concentrations of RNA using both assays. RT-qPCR shows higher frequency of detection of N1 and N2 gene regions in PCS (92.7, 90.6%, n = 6) as compared to PGS samples (79.2, 82.3%, n = 5). Sampling of PCS may therefore be an effective approach for SARS-CoV-2 viral quantification, especially during periods of declining and low COVID-19 incidence in the community. The pepper mild mottle virus (PMMoV) is determined to have a less variable RNA signal in PCS over a three month period for two WRRFs, regardless of environmental conditions, compared to Bacteroides 16S rRNA or human 18S rRNA, making PMMoV a potentially useful biomarker for normalization of SARS-CoV-2 signal. PMMoV-normalized PCS RNA signal from WRRFs of two cities correlated with the regional public health epidemiological metrics, identifying PCS normalized to a fecal indicator (PMMoV) as a potentially effective tool for monitoring trends during decreasing and low-incidence of infection of SARS-Cov-2 in communities.</div></div><div data-prj=prj46><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.scitotenv.2021.145319 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/G21-262001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-262001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-262001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-262001/>Catching a resurgence: Increase in SARS-CoV-2 viral RNA identified in wastewater 48 h before COVID-19 clinical tests and 96 h before hospitalizations</a></strong><br><a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=uni0027>Patrick M. D’Aoust</a>,
<a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=uni0117>Tyson E. Graber</a>,
<a href=/gwf-publications/people/%C3%89/%C3%89lisabeth-Mercier/ data-uni=uni0027>Élisabeth Mercier</a>,
<a href=/gwf-publications/people/D/Danika-Montpetit/ data-uni=uni0004>Danika Montpetit</a>,
<a href=/gwf-publications/people/I/Ilya-Alexandrov/ data-uni=uni0802>Ilya Alexandrov</a>,
<a href=/gwf-publications/people/N/Nafisa-Neault/ data-uni=uni0027>Nafisa Neault</a>,
<a href=/gwf-publications/people/A/Aiman-Tariq-Baig/ data-uni=uni0117>Aiman Tariq Baig</a>,
<a href=/gwf-publications/people/J/Janice-Mayne/ data-uni=uni0027>Janice Mayne</a>,
<a href=/gwf-publications/people/X/Xu-Zhang/ data-uni=uni0576>Xu Zhang</a>,
<a href=/gwf-publications/people/T/Tommy-Alain/ data-uni=uni0027>Tommy Alain</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/N/Nivetha-Srikanthan/ data-uni=uni0002>Nivetha Srikanthan</a>,
<a href=/gwf-publications/people/M/Malcolm-R-MacKenzie/ data-uni=uni0022>Malcolm R. MacKenzie</a>,
<a href=/gwf-publications/people/D/Daniel-Figeys/ data-uni=uni0027>Daniel Figeys</a>,
<a href=/gwf-publications/people/M/Manuel-Dujovny/ data-uni=uni0602>Manuel Dujovny</a>,
<a href=/gwf-publications/people/P/Peter-J%C3%BCni/ data-uni=uni0124>Peter Jüni</a>,
<a href=/gwf-publications/people/A/Alex-MacKenzie/ data-uni=uni0117>Alex MacKenzie</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0027>Robert Delatolla</a><br><a href=/gwf-publications/volumes/G21-262/ class=text-muted>Science of The Total Environment, Volume 770</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-262001><div class="card-body p-3 small">Curtailing the Spring 2020 COVID-19 surge required sweeping and stringent interventions by governments across the world. Wastewater-based COVID-19 epidemiology programs have been initiated in many countries to provide public health agencies with a complementary disease tracking metric and non-discriminating surveillance tool. However, their efficacy in prospectively capturing resurgences following a period of low prevalence is unclear. In this study, the SARS-CoV-2 viral signal was measured in primary clarified sludge harvested every two days at the City of Ottawa's water resource recovery facility during the summer of 2020, when clinical testing recorded daily percent positivity below 1%. In late July, increases of >400% in normalized SARS-CoV-2 RNA signal in wastewater were identified 48 h prior to reported >300% increases in positive cases that were retrospectively attributed to community-acquired infections. During this resurgence period, SARS-CoV-2 RNA signal in wastewater preceded the reported >160% increase in community hospitalizations by approximately 96 h. This study supports wastewater-based COVID-19 surveillance of populations in augmenting the efficacy of diagnostic testing, which can suffer from sampling biases or timely reporting as in the case of hospitalization census.</div></div></div><div class=col-lg-3><a class="btn btn-lg btn-secondary btn-block mb-2" href="https://www.semanticscholar.org/search?q=Patrick+M.+D%E2%80%99Aoust" title="Search for 'Patrick M. D’Aoust' on Semantic Scholar"><i class="ai ai-semantic-scholar"></i><span class=pl-sm-2>Search</span></a><div class=row><div class="col-12 col-md-6 col-lg-12"><div class=card><h5 class=card-header>Co-authors</h5><ul class="list-group list-group-flush list-group-compact"><li class=list-group-item><a href=/gwf-publications/people/R/Robert-Delatolla/ class=align-middle data-uni=uni0027>Robert Delatolla</a>
<span class="badge badge-secondary align-middle ml-2">6</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mark-R-Servos/ class=align-middle data-uni=uni0002>Mark R. Servos</a>
<span class="badge badge-secondary align-middle ml-2">6</span></li><li class=list-group-item><a href=/gwf-publications/people/%C3%89/%C3%89lisabeth-Mercier/ class=align-middle data-uni=uni0027>Élisabeth Mercier</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/T/Tyson-E-Graber/ class=align-middle data-uni=uni0117>Tyson E. Graber</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/X/Xiaoli-Pang/ class=align-middle data-uni=uni0376>Xiaoli Pang</a>
<span class="badge badge-secondary align-middle ml-2">3</span></li><li class="list-group-item list-group-toggle-btn py-1" data-toggle=collapse data-target=#more-coauthors aria-expanded=false aria-controls=more-coauthors>show all...</li><div class="collapse border-top" id=more-coauthors><li class=list-group-item><a href=/gwf-publications/people/A/Alex-MacKenzie/ class=align-middle data-uni=uni0117>Alex MacKenzie</a>
<span class="badge badge-secondary align-middle ml-2">3</span></li><li class=list-group-item><a href=/gwf-publications/people/T/Tommy-Alain/ class=align-middle data-uni=uni0027>Tommy Alain</a>
<span class="badge badge-secondary align-middle ml-2">3</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Alex-H-S-Chik/ class=align-middle data-uni=uni0451>Alex H. S. Chik</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Melissa-B-Glier/ class=align-middle data-uni=uni0771>Melissa B. Glier</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Chand-Mangat/ class=align-middle data-uni=uni0107>Chand Mangat</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Natalie-Prystajecky/ class=align-middle data-uni=uni0924>Natalie Prystajecky</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yuanyuan-Qiu/ class=align-middle data-uni=uni0509>Yuanyuan Qiu</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/X/Xin-Tian/ class=align-middle data-uni=uni0103>Xin Tian</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Syeda-Tasneem-Towhid/ class=align-middle data-uni=uni0027>Syeda Tasneem Towhid</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Meghan-Fuzzen/ class=align-middle data-uni=uni0002>Meghan Fuzzen</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Nivetha-Srikanthan/ class=align-middle data-uni=uni0002>Nivetha Srikanthan</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/D/Danika-Montpetit/ class=align-middle data-uni=uni0004>Danika Montpetit</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/I/Ilya-Alexandrov/ class=align-middle data-uni=uni0802>Ilya Alexandrov</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Nafisa-Neault/ class=align-middle data-uni=uni0027>Nafisa Neault</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Aiman-Tariq-Baig/ class=align-middle data-uni=uni0117>Aiman Tariq Baig</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Janice-Mayne/ class=align-middle data-uni=uni0027>Janice Mayne</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/X/Xu-Zhang/ class=align-middle data-uni=uni0576>Xu Zhang</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Malcolm-R-MacKenzie/ class=align-middle data-uni=uni0022>Malcolm R. MacKenzie</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/D/Daniel-Figeys/ class=align-middle data-uni=uni0027>Daniel Figeys</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Philip-J-Schmidt/ class=align-middle data-uni=uni0002>Philip J. Schmidt</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Nicole-Acosta/ class=align-middle data-uni=uni0007>Nicole Acosta</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Hadi-A-Dhiyebi/ class=align-middle data-uni=uni0002>Hadi A. Dhiyebi</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Casey-R-J-Hubert/ class=align-middle data-uni=uni0007>Casey R. J. Hubert</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jennifer-Kopetzky/ class=align-middle data-uni=uni0196>Jennifer Kopetzky</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Shelley-Peterson/ class=align-middle data-uni=uni0107>Shelley Peterson</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Monica-B-Emelko/ class=align-middle data-uni=uni0002>Monica B. Emelko</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Amy-Xiao/ class=align-middle data-uni=uni0246>Amy Xiao</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Nada-Hegazy/ class=align-middle data-uni=uni0027>Nada Hegazy</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jianjun-Jia/ class=align-middle data-uni=uni0212>Jianjun Jia</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Shungang-Wan/ class=align-middle data-uni=uni0237>Shungang Wan</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Md-Pervez-Kabir/ class=align-middle data-uni=uni0027>Md Pervez Kabir</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/W/Wanting-Fang/ class=align-middle data-uni=uni0027>Wanting Fang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Maria-E-Hasing/ class=align-middle data-uni=uni0012>Maria E. Hasing</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Minqing-Ivy-Yang/ class=align-middle data-uni=uni0022>Minqing Ivy Yang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jianxian-Sun/ class=align-middle data-uni=uni0022>Jianxian Sun</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Julio-PlazaD%C3%ADaz/ class=align-middle data-uni=uni1074>Julio Plaza‐Díaz</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Z/Zhihao-Zhang/ class=align-middle data-uni=uni1075>Zhihao Zhang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Aaron-Cowan/ class=align-middle data-uni=uni0027>Aaron Cowan</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/W/Walaa-Eid/ class=align-middle data-uni=uni0117>Walaa Eid</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Sean-E-Stephenson/ class=align-middle data-uni=uni0117>Sean E. Stephenson</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Matthew-J-Wade/ class=align-middle data-uni=uni0398>Matthew J. Wade</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Hui-Peng/ class=align-middle data-uni=uni0022>Hui Peng</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/E/Elizabeth-A-Edwards/ class=align-middle data-uni=uni0022>Elizabeth A. Edwards</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/E/Eric-J-Alm/ class=align-middle data-uni=uni0246>Eric J. Alm</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jean-Baptiste-Burnet/ class=align-middle data-uni=uni0239>Jean-Baptiste Burnet</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Sarah-Dorner/ class=align-middle data-uni=uni0239>Sarah Dorner</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Q/Qiudi-Geng/ class=align-middle data-uni=uni0153>Qiudi Geng</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/John-P-Giesy/ class=align-middle data-uni=uni0016>John P. Giesy</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/R/R-Michael-L-McKay/ class=align-middle data-uni=uni0153>R. Michael L. McKay</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Michael-R-Mulvey/ class=align-middle data-uni=uni0107>Michael R. Mulvey</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yuwei-Xie/ class=align-middle data-uni=uni0050>Yuwei Xie</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/B/Bernadette-Conant/ class=align-middle data-uni=uni1570>Bernadette Conant</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Steve-E-Hrudey/ class=align-middle data-uni=uni0012>Steve E. Hrudey</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Kamya-Bhatnagar/ class=align-middle data-uni=uni0027>Kamya Bhatnagar</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Huy-Dung-Hoang/ class=align-middle data-uni=uni0117>Huy-Dung Hoang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Julio-Plaza-Diaz/ class=align-middle data-uni=uni0117>Julio Plaza Diaz</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Ainslie-Butler/ class=align-middle data-uni=uni1578>Ainslie Butler</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/L/Lawrence-Goodridge/ class=align-middle data-uni=uni0009>Lawrence Goodridge</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jian-Jun-Jia/ class=align-middle data-uni=uni0027>Jian Jun Jia</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Marc-Andr%C3%A9-Langlois/ class=align-middle data-uni=uni0602>Marc André Langlois</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Manuel-Dujovny/ class=align-middle data-uni=uni0602>Manuel Dujovny</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Peter-J%C3%BCni/ class=align-middle data-uni=uni0124>Peter Jüni</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li></div></ul></div></div><div class="col-12 col-md-6 col-lg-12"><div class="card my-2 my-md-0 my-lg-2"><h5 class=card-header>Venues</h5><ul class="list-group list-group-flush list-group-compact"><li class=list-group-item><a href=/gwf-publications/venues/gwf/ class=align-middle>GWF</a><span class="badge badge-secondary align-middle ml-2">6</span></li></ul></div></div></div></div></div></section></div><footer class="bg-gradient-light py-2 py-xl-3 mt-3 mt-md-4 mt-xl-5"><div class=container><p class="text-muted small px-1">Global Water Futures Publications!</p></div></footer><script src=https://code.jquery.com/jquery-3.3.1.slim.min.js integrity=sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js integrity=sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut crossorigin=anonymous></script><script src=https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js integrity=sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k crossorigin=anonymous></script><script>$(function(){$('[data-toggle="tooltip"]').tooltip(),$("#toggle-all-abstracts")&&($("#toggle-all-abstracts").click(function(){var e=$("#toggle-all-abstracts");e.attr("disabled",!0),e.attr("data-toggle-state")=="hide"?($(".abstract-collapse").collapse("show"),e.attr("data-toggle-state","show")):($(".abstract-collapse").collapse("hide"),e.attr("data-toggle-state","hide")),e.attr("disabled",!1)}),$("#toggle-all-abstracts").attr("disabled",!1))})</script></body></html>